pyproject.toml

```
@@ -0,0 +1,132 @@
+[project]
+name = "janus-api"
+version = "0.0.1"
+description = "Tools for machine learnt interatomic potentials"
+authors = [
+    { name = "Cameron Ball" },
+]
+requires-python = ">=3.10"
+classifiers = [
+    "Programming Language :: Python",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Natural Language :: English",
+    "Development Status :: 3 - Alpha",
+]
+readme = "README.md"
+
+dependencies = [
+    "janus-core",
+    "fastapi<1.0.0,>=0.95.0",
+    "uvicorn<1.0.0,>=0.22.0",
+    "python-multipart",
+    "fastapi-cors"
+]
+
+[project.optional-dependencies]
+
+[project.scripts]
+
+[project.urls]
+repository = "https://github.com/stfc/janus-core/"
+documentation = "https://stfc.github.io/janus-core/"
+
+[dependency-groups]
+dev = [
+    "coverage[toml]<8.0.0,>=7.4.1",
+    "pgtest<2.0.0,>=1.3.2",
+    "pytest<9.0,>=8.0",
+    "pytest-cov<5.0.0,>=4.1.0",
+    "tox-uv<2.0,>=1.16.1",
+    "wheel<1.0,>=0.42",
+]
+
+docs = [
+    "furo<2025.0.0,>=2024.1.29",
+    "markupsafe<2.1",
+    "numpydoc<2.0.0,>=1.6.0",
+    "sphinx<8.0.0,>=7.2.6",
+    "sphinxcontrib-contentui<1.0.0,>=0.2.5",
+    "sphinxcontrib-details-directive<1.0,>=0.1",
+    "sphinx-copybutton<1.0.0,>=0.5.2",
+]
+
+pre-commit = [
+    "pre-commit<4.0.0,>=3.6.0",
+    "ruff<1.0.0,>=0.7.4",
+]
+
+[build-system]
+requires = ["pdm-backend"]
+build-backend = "pdm.backend"
+
+[tool.pytest.ini_options]
+# Configuration for pytest
+python_files = "test_*.py"
+addopts = '--cov-report xml'
+pythonpath = ["."]
+
+[tool.coverage.run]
+# Configuration of coverage.py
+# reporting which lines of your plugin are covered by tests
+source=["janus_core"]
```

```suggestion
source=["janus_web"]
```
or wherever the main codebase lives